### PR TITLE
add snarkyjs-pack community package to README.md [now with provable tests and doc comments!]

### PR DESCRIPTION
**Pull request by barriebyron**
This PR adds our second community package for o1js to the README 

https://discord.com/channels/484437221055922177/915745847692636181/1131617947534753962

### CODE REVIEW

**Review by ymekuria — APPROVED** [review]
Exciting to see more community packages!

**Review by mitschabaude — CHANGES_REQUESTED** [review]
Just making sure -- someone review the library?

**Comment by Trivo25** [comment]
I just took a quick look. It would be good if the package could add an e2e integration test (demonstrating the functionality in a smart contract, or other provable code). Additionally, I couldn't find any doc comments (which would be a nice to have!) cc @45930

**Comment by 45930** [comment]
@Trivo25 yeah I will plan to do that after my cohort 1 presentation.  I am busy working on that for the next couple weeks.

**Comment by 45930** [comment]
@mitschabaude no one has reviewed thoroughly AFAIK

**Comment by Trivo25** [comment]
> @Trivo25 yeah I will plan to do that after my cohort 1 presentation. I am busy working on that for the next couple weeks.

That's fine! It's not a must, but would be great *if* you want to be listed as a community package in SnarkyJS :)

**Review by mitschabaude — APPROVED** [review]

**Comment by barriebyron** [comment]
@mitschabaude I see you approved, because I am uncertain about the technical status, would you be up for merging this PR when you feel the package meets the high quality package guidelines? Another option, we label it as experimental?

**Comment by mitschabaude** [comment]
> @mitschabaude I see you approved, because I am uncertain about the technical status, would you be up for merging this PR when you feel the package meets the high quality package guidelines? Another option, we label it as experimental?

I only approved to no longer be "requesting changes" 😅 I think the comment by Florian still makes sense, we want those packages to have a test

**Comment by barriebyron** [comment]
Excited about our second package candidate. We'll leave this PR open as we await package updates from @45930

**Comment by 45930** [comment]
Hi guys.  I added provable tests and doc comments, as well as rebranded everything to "o1js".

https://github.com/45930/o1js-pack/blob/main/src/lib/PackingPlant.ts
https://github.com/45930/o1js-pack/blob/main/tests/provable/end_to_end.test.ts

**Comment by barriebyron** [comment]
ready to merge, thank you @45930 for your contribution and community package 

I contributed https://github.com/45930/o1js-pack/pull/14

File: README.md [hunk 1]
```diff
@@ -47,5 +47,6 @@ See the [Contributing guidelines](https://github.com/o1-labs/o1js/blob/main/CONT
 High-quality community packages from open source developers are available for your project.
 
 - **o1js-elgamal** A partially homomorphic encryption library for o1js based on Elgamal encryption: [GitHub](https://github.com/Trivo25/o1js-elgamal) and [npm](https://www.npmjs.com/package/o1js-elgamal)
+- **snarkyjs-pack** A library for SnarkyJS that allows a zkApp developer to pack extra data into a single Field. [GitHub](https://github.com/45930/o1js-pack) and [npm](https://www.npmjs.com/package/o1js-pack)
```
Comment (barriebyron) [inline README.md:50]:
```suggestion
- **o1js-pack** A library for o1js that allows a zkApp developer to pack extra data into a single Field. [GitHub](https://github.com/45930/o1js-pack) and [npm](https://www.npmjs.com/package/o1js-pack)
```

**Pull request merged**